frontend/GoodGrocer/test/Login.test.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "@testing-library/jest-dom";
```

Comment:
```suggestion
``` 
Just so that it's less confusing when we do more unit tests, I think we should remove this import fo now since we only added it when we were testing toBeOnTheScreen